bindings/nodejs/lib/types/utils/type_aliases.ts

```diff
@@ -0,0 +1,4 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+export type u16 = number;
```

Comment:
Maybe it's good to indicate that it's `u16` to TS users? or we may just use `number` 🤷🏻‍♂️